CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ include (${SPARTA_BASE}/cmake/sparta-config.cmake)
 # Set up STF library
 set (STF_LIB_BASE ${PROJECT_SOURCE_DIR}/stf_lib)
 set (DISABLE_STF_DOXYGEN ON)
-include(${STF_LIB_BASE}/cmake/stf-config.cmake)
+#include(${STF_LIB_BASE}/cmake/stf-config.cmake)
```

Comment:
Are we not building stf_lib as a part of this flow?

Reply:
We still are -- it's included below.  We just don't need STF-specific library definitions right now (they overlap with Sparta).

Reply:
got it. thank you.